cars/v1/vanilla/templates/config/jvm.options

```diff
@@ -34,7 +34,7 @@
 
 ## GC configuration
 {# The implicit default is to use the default GC (depending on the JDK version) #}
-{%- if use_cms_gc is not defined and use_g1_gc is not defined and use_parallel_gc is not defined %}
+{%- if use_cms_gc is not defined and use_g1_gc is not defined and use_parallel_gc is not defined and use_zgc is not defined %}
```

Comment:
nit: This is out of scope for this PR, but we should use Jinja if/elif/else instead of listing all supported GCs in this line.

(This was part of my initial review but GitHub ate it)